app/controllers/general.php

```diff
@@ -485,6 +485,10 @@
                     $error->setType(AppwriteException::GENERAL_ROUTE_NOT_FOUND);
                     break;
             }
+        } elseif ($error instanceof Utopia\Database\Exception\Conflict) {
+            $error = new AppwriteException(AppwriteException::DOCUMENT_UPDATE_CONFLICT, null, null, $error);
```

Comment:
I think this can just be:

```suggestion
            $error = new AppwriteException(AppwriteException::DOCUMENT_UPDATE_CONFLICT);
```

Reply:
I wanted to make sure to pass the underlying error through as well.